apps/ui/components/single-search/SearchForm.tsx

```diff
@@ -86,15 +87,23 @@ const Filters = styled.div<{ $areFiltersVisible: boolean }>`
 
   @media (min-width: ${breakpoints.m}) {
     margin-top: var(--spacing-s);
-    grid-template-columns: 1fr 1fr;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-columns: repeat(3, auto);
+    > div {
+      grid-column: span 1;
+    }
   }
 
   @media (min-width: ${breakpoints.l}) {
-    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-columns: repeat(3, 1fr);
   }
 `;
 
-const StyledSelect = styled(Select<OptionType>)`
+const Filters = styled.div<{ children: ReactNode[] }>`
+  ${filterGrid}
+`;
+
+const StyledSelect = styled(Select<OptionType>)<{ name?: string }>`
```

Comment:
Name is only used in the `<StyledSelect>` element. Using a `name` attribute in the element results in errors, as `StyledSelect` doesn't otherwise expect it.

Reply:
It shouldn't require it. If `name` is defined for HDS Select the prop should be passed through when inheriting with styled. To me it seems like `name` is not defined for Select (and it's not), adding it just hides the typescript error without doing anything.

Reply:
Yes... "Doing this properly would be to code a common Dropdown-component of our own from scratch, which wraps a HDS Select to use it properly with react-hook-form (like I did with TextInput)."